world/mujoco_ros/include/mujoco_ros/defs.h

```diff
@@ -7,8 +7,43 @@
 #include <fstream>
 #include <strstream>
 #include <iomanip>
+#include <regex>
+
 #include "GLFW/glfw3.h"
 #include "ros/ros.h"
 #include "mujoco/mujoco.h"
 #include "std_msgs/Empty.h"
 #include "mujoco_ros/SensorDataStamped.h"
+
+#define PRX_DEBUG_PRINT std::cout << __PRETTY_FUNCTION__ << ": " << __LINE__ << std::endl;
```

Comment:
Nit: Do we want to use `PRX_` prefixs in this repository?

Reply:
I copy/pasted this from ML4KP. I don't have a preference.